Applications/MaterialAssist/Client/MaterialAssistClientBoards.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System.Text;
+
+using HomagConnect.Base.Services;
+using HomagConnect.MaterialAssist.Contracts.Base;
+using HomagConnect.MaterialAssist.Contracts.Boards;
+using HomagConnect.MaterialAssist.Contracts.Boards.Interfaces;
+using HomagConnect.MaterialManager.Contracts.Material.Boards;
+
+namespace HomagConnect.MaterialAssist.Client
+{
+    /// <summary>
+    /// materialAssist client for boards
+    /// </summary>
+    public class MaterialAssistClientBoards : ServiceBase, IMaterialAssistClientBoards
+    {
+        /// <inheritdoc />
+        public MaterialAssistClientBoards(HttpClient client) : base(client) { }
+
+        #region Delete
+
+        /// <inheritdoc />
+        public async Task DeleteBoardById(string id)
```

Comment:
Should we have something as return type beside a the usual status code which will be 200?